srv/components/system/files/confstore_template.j2

```diff
@@ -77,4 +77,21 @@ server_node>{{ machine_id }}>storage_set_id:{{ pillar['cluster'][node]['storage_
 server_node>{{ machine_id }}>cluster_id:{{ cluster_id }}
 server_node>{{ machine_id }}>s3_instances:{{ pillar['cluster'][node]['s3_instances'] }}
 {%- endfor %}
-
+{%- for enclosure in pillar['storage'].keys() %}
+storage_enclosure>{{ pillar['storage'][enclosure]['enclosure_id'] }}>name:{{ enclosure }}
+storage_enclosure>{{ pillar['storage'][enclosure]['enclosure_id'] }}>type:{{ pillar['storage'][enclosure]['type'] }}
```

Comment:
@sradhanand-pati shouldn't it be 
```
pillar['storage'][enclosure]['enclosure_id']['type']
```
and the same for similar line below.

Reply:
No, 'type' is under 'enclosure' not under 'enclosure_id'.

Reply:
@andkononykhin2 
`pillar['storage'][enclosure]['type']` & `pillar['storage'][enclosure]['enclosure_id']` are at same level in our pillar:
```
storage:
    enclosure-1:
      type: RBOD                      # RBOD/JBOD/Virtual/Other
      controller:
        type:
        primary:
          ip: 10.0.0.2
          port: 80
        secondary:
          ip: 10.0.0.3
          port: 80
        user: manage
        secret:
```

Reply:
Got it. Thank you @83bhp @ypise